content/blog/2020/10/camel-quarkus-release-1.2.0-and-1.3.0/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Camel Quarkus 1.2.0 and 1.3.0 Released"
+date: 2020-10-20
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 1.2.0 and 1.3.0 Released"
+summary: "The highlights of Camel Quarkus 1.2.0 and 1.3.0"
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
+
+## Why two releases one after another?
+
+Camel Quarkus 1.2.0 was scheduled - with some reasonable time reserve - to be included in the Quarkus Platform release
+1.9.0. The next day after we opened the voting, we realized that Camel 3.6.0 is going to be released in the coming days
+and that it could actually catch Quarkus 1.9.0 too. Therefore we did one more release upgrading to Camel 3.6.0, so that
+the freshly released [Quarkus Platform 1.9.0](https://quarkus.io/blog/quarkus-1-9-0-final-released/) contains the newest
+and greatest Camel.
+
+## What's new
+
+First of all, Camel 3.6.0 alone brings a lot of
+[improvements and optimizations](https://camel.apache.org/blog/2020/10/Camel36-Whatsnew/) that make Camel Quarkus
```

Comment:
```suggestion
[improvements and optimizations](https://camel.apache.org/blog/2020/10/Camel36-Whatsnew/) that makes Camel Quarkus
```

Reply:
Oh, really? My parser of the English language says that "make" relates to "improvements and optimizations" and hence it has to be in plural (without "s"). "Makes" would be possible if it hanged on "Camel 3.6.0" - do you really understand the sentence in that way? I am truly baffled.

Reply:
I've read it over a few times now. Yeah, the original wording is ok.

That's what happens when you do reviews before consuming the first coffee of the day :coffee:

Reply:
Thank you for your effort!